.github/workflows/publish-to-pypi.yml

```
@@ -2,15 +2,11 @@ name: Publish to PyPI
 
 on:
   release:
-    types: [released]
+    types: [ released ]
 
 jobs:
   publish:
-    uses: spacetelescope/action-publish_to_pypi/.github/workflows/workflow.yml@master
-    with:
-      test: false
-      build_platform_wheels: false # Set to true if your package contains a C extension
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish_pure_python.yml@v1
```

To not upload to PyPi for non-releases:
```suggestion
    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish_pure_python.yml@v1
    with:
      upload_to_pypi: ${{ gitub.event_name == 'released'}}
```
Then the upload will only be triggered when the action is triggered by releases.

---

wow, powerful options!